src/core/usecases/launcher.ts

```diff
@@ -169,6 +170,15 @@ export declare namespace LauncherState {
     );
 }
 
+export type K8sClusterParams = {
+    "K8S_CLUSTER": string;
+    "K8S_USER": string;
+    "K8S_SERVER_URL": string;
+    "K8S_NAMESPACE": string;
+    "K8S_TOKEN": string;
+    "K8S_EXPIRATION": number;
+};
+
```

Comment:
Theses are ment to be values used to replace default in the form.  
These are the values we made avalible to the catalog maintainders:  
https://github.com/InseeFrLab/onyxia-web/blob/cd1cbbeb7f2c30f9ec226890c8a19a7c57e40a1b/src/core/ports/OnyxiaApiClient.ts#L154-L210  

No token, no expiration.